lib/metadataTypes/definitions/Journey.definition.js

```diff
@@ -716,6 +717,18 @@ export default {
         'activities[].schema': {
             skipValidation: true,
         },
+        'activities[].arguments.activityData.updateContactFields[].r__dataExtension_key': {
+            isCreateable: false,
+            isUpdateable: false,
+            retrieving: true,
+            template: true,
+        },
+        'activities[].arguments.activityData.updateContactFields[].field': {
+            isCreateable: false,
+            isUpdateable: false,
```

Comment:
```suggestion
            isCreateable: true,
            isUpdateable: true,
```
this one you DO want creatable and updatable because it's a standard field.

please also add the new custom field i suggested (r__dataExtensionField_name)